examples/bpk-component-card-list/examples.tsx

```diff
@@ -152,7 +151,6 @@ const snippetProps = {
 };
 
 const DestinationCard = (i: number) => (
-  // Usage Suggestion: define minWidth from consumer side by using className
```

Comment:
The Card width is determined by the number of cards set to be displayed ([desktop](https://github.com/Skyscanner/backpack/pull/3878/files#diff-49825811eb055632d14400d08a33814bc582358ef6b4b5f9ed8ec089c1ca131aL41), [mobile](https://github.com/Skyscanner/backpack/pull/3878/files#diff-49825811eb055632d14400d08a33814bc582358ef6b4b5f9ed8ec089c1ca131aR73)), so the minWidth cannot be specified, otherwise it will [conflict](https://github.com/Skyscanner/backpack/pull/3878/files#diff-6a0e148934bce32a7b29700751ad5e7d3eabc660d11743cf2229efcd522e159eR51) with `initiallyShownCardsDesktop`/`initiallyShownCardsMobile`.